google/resource_sql_database.go

```diff
@@ -86,6 +90,13 @@ func resourceSqlDatabaseRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
+	// In the import case this won't be set
+	if _, ok := d.GetOk("instance"); !ok {
```

Comment:
Makes sense however for it to work I've had to add a `d.setId` in the Create method as it's currently set as part of the Read.